ClientApp/viking-vault/src/UserDashboard/AccountPage.tsx

```diff
@@ -261,24 +261,22 @@ class AccountPage extends React.Component<any, IAccountPageState>{
 
     render(){
         return(
```

Comment:
Way too much logic is done inside the JSX itself, try moving some of the ternary operators outside of it. (as mentioned below for the nested ternary operators)

Reply:
Done